WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift

```diff
@@ -169,20 +165,25 @@ struct BlazeCampaignCreationForm: View {
             isShowingAISuggestionsErrorAlert = newValue == .failedToLoadAISuggestions
         }
         .alert(isPresented: $isShowingAISuggestionsErrorAlert, content: {
-            Alert(title: Text(Localization.ErrorAlert.title),
-                  message: Text(Localization.ErrorAlert.ErrorMessage.fetchingAISuggestions),
-                  primaryButton: .default(Text(Localization.ErrorAlert.retry), action: {
+            Alert(title: Text(Localization.AISuggestionsErrorAlert.title),
+                  message: Text(Localization.AISuggestionsErrorAlert.ErrorMessage.fetchingAISuggestions),
+                  primaryButton: .default(Text(Localization.AISuggestionsErrorAlert.retry), action: {
                 Task {
                     await viewModel.loadAISuggestions()
                 }
             }),
                   secondaryButton: .cancel())
         })
-        .task {
-            await viewModel.loadAISuggestions()
+        .alert(isPresented: $viewModel.isShowingMissingImageErrorAlert, content: {
+            Alert(title: Text(Localization.NoImageErrorAlert.noImageFound),
+                  dismissButton: .default(Text(Localization.NoImageErrorAlert.ok)))
+        })
```

Comment:
This modifier is deprecated, please use `.alert(_ titleKey:isPresented:action)` instead.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/11771/commits/6f182a7b573217541465ed199ff3be840ef33ba6 and https://github.com/woocommerce/woocommerce-ios/pull/11771/commits/6f267f0389ef9ba470ff4bf3671d3060a81d61f7